crypten/mpc/mpc.py

```diff
@@ -753,7 +764,14 @@ def max_pool2d(self, kernel_size, padding=None, stride=None, return_indices=Fals
 
     @mode(Ptype.arithmetic)
     def _max_pool2d_backward(
-        self, indices, kernel_size, padding=None, stride=None, output_size=None
+        self,
+        indices,
+        kernel_size,
+        padding=None,
```

Comment:
Q: Can't the padding be set here to 0 (by default), in that case, there is no need to check if the padding is ```None```